src/fr-command-lrzip.c

```diff
@@ -76,7 +81,7 @@ list__process_line (char     *line,
 	fdata->dir = FALSE;
 	fdata->link = NULL;
 
-	if (fdata->name == 0)
+	if (strlen (fdata->name) == 0)
```

Comment:
what about the faster:
```suggestion
	if (*fdata->name == '\0')
```

Reply:
done